### PR TITLE
Drop 0.4 and update a few deprecated syntax

### PR DESCRIPTION
**Pull request by musm**

### CODE REVIEW

File: src/LibExpat.jl [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
 using Compat
-using Compat: String, unsafe_string
 
 import Base: getindex, show, parse
 
-if is_windows() 
+if Compat.Sys.iswindows() 
     const libexpat = "libexpat-1"
-elseif is_unix() 
+elseif Compat.Sys.islinux()
```
Comment (tkelman) [inline src/LibExpat.jl]: any unix, not just linux

Comment (musm) [inline src/LibExpat.jl]: thanks, good catch.

Comment (musm) [inline src/LibExpat.jl]: Now if this had a BinDeps then these would be library aliases but it does not since libexpat is shipped with julia.

**Comment by musm** [comment]
I wonder if it would be just better to drop 0.5 as well ?

**Comment by tkelman** [comment]
What for?

**Comment by musm** [comment]
True, probably best to wait a bit longer for that

**Pull request merged**